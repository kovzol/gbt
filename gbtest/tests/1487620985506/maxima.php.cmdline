#!/bin/sh
cd tests/1487620985506
cat maxima.php.in | /usr/bin/timeout -k 120 60 /usr/bin/time /usr/bin/maxima > maxima.php.out 2>maxima.php.err