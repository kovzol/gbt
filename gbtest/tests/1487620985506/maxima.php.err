3.94user 0.30system 0:04.26elapsed 99%CPU (0avgtext+0avgdata 1456580maxresident)k
0inputs+480outputs (0major+5456minor)pagefaults 0swaps
