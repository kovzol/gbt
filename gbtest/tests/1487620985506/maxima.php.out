
Maxima 5.37.2 http://maxima.sourceforge.net
using Lisp GNU Common Lisp (GCL) GCL 2.6.12
Distributed under the GNU Public License. See the file COPYING.
Dedicated to the memory of William Schelter.
The function bug_report() provides bug reporting information.
(%i1) (%o3) [(- 50 v1) - 63, 113 - 25 v3, 13 - 25 v4, 13 - 25 v2, 
                  2             2
    (- 62500 v5 v6 ) + 101875 v6  + 65000 v5 v6 - 105950 v6 - 16900 v5 + 27547]
(%i4) 